test/server/cards/02-AoA/AmberConductionUnit.spec.js

```diff
@@ -46,4 +46,46 @@ describe('AmberConductionUnit', function () {
             expect(this.grenadeSnib.stunned).toBe(false);
         });
     });
+
+    describe("Amber Conduction Unit's ability outside of the main phase", function () {
+        beforeEach(function () {
+            this.setupTest({
+                player1: {
+                    house: 'logos',
+                    hand: ['jargogle', 'ghosthawk', 'daughter', 'brillix-ponder', 'strange-gizmo'],
+                    amber: 6
+                },
+                player2: {
+                    inPlay: ['æmber-conduction-unit'],
+                    amber: 0
+                }
+            });
+        });
+
+        it('should count creatures reaping during start of turn', function () {
+            this.player1.play(this.jargogle);
+            this.player1.clickCard(this.ghosthawk);
+            this.player1.play(this.brillixPonder);
+            this.player1.play(this.daughter);
+            this.player1.play(this.strangeGizmo);
+            this.brillixPonder.tokens.ward = 1;
+            this.daughter.tokens.ward = 1;
+            this.player1.endTurn();
+            this.player2.clickPrompt('mars');
+            this.player2.endTurn();
+
+            // Strange Gizmo causes Jargogle to play ghosthawk after forging a key
+            this.player1.clickPrompt('red');
+            this.player1.clickPrompt('deploy right');
+            this.player1.clickCard(this.brillixPonder);
+            this.player1.clickCard(this.brillixPonder);
+            this.player1.clickCard(this.daughter);
+            this.player1.clickPrompt('logos');
+            expect(this.brillixPonder.location).toBe('play area');
+            expect(this.brillixPonder.tokens.stun).toBe(1);
+            expect(this.daughter.location).toBe('play area');
+            expect(this.daughter.tokens.stun).toBe(undefined);
+            expect(this.player1).toHavePrompt('Choose a card to play, discard or use');
```

Comment:
Dunno if it’s worth doing another reap here now that we’re in the main phase to show that it wouldn’t get stunned because Brillix Ponder’s stun counted for the entire turn.